src/System.Management.Automation/engine/GetCommandCommand.cs

```diff
@@ -319,7 +314,7 @@ public PSTypeName[] ParameterType
             {
                 if (value == null)
                 {
-                    throw new ArgumentNullException("value");
+                    throw new ArgumentNullException(nameof(value));
```

Comment:
cc: @iSazonov 

Does specifying the `paramName` add any (ahem) value. This current code results in the following message:

```
Value cannot be null.
Parameter name: value
```

But instead `ArgumentNullException()` could be used for a less verbose message:
```
Value cannot be null.
```